src/app/shared/components/template/components/points-item/points-item.component.ts

```diff
@@ -65,7 +64,7 @@ export class TmplParentPointBoxComponent
     this.animCelebrationOptions = {
       path: getImageAssetPath("/plh_lottie/habits/cascading_stars.json"),
       name: "celebration",
-      autoplay: false,
+      autoplay: true,
       loop: false,
       rendererSettings: {
         // svg scaling options: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio
```

Comment:
@chrismclarke Presumably to solve Esmee's issue of the Stars being behind we just need to bring them to the front. I don't know exactly where but guess that it's just a question of inserting z-index: 999, or something in the right place. is that here?
```suggestion
        // svg scaling options: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio
        z-index: 999,
```

Reply:
You're right - previously the code had a toggle where it added an extra class when the play started which updated the z-index. Now that the element only exists this is not necessary, but I hadn't made 'play' state z-index the default

Fixed with fb30e7caffb09abbf8d805211415b79d7de19c06